test/intl.mjs

```diff
@@ -600,8 +599,7 @@ describe('Intl', () => {
         });
         const withValues = getValues('with');
         const withErrorExpected =
-          withValues === RangeError ||
-          ((nodeVersion === '14' || nodeVersion === '12') && withValues.nodeBefore15 === RangeError);
+          withValues === RangeError || (nodeVersion === '12' && withValues.node12 === RangeError);
```

Comment:
Note: these we can probably eliminate if we're not supporting Node 12 anymore, but it's fine to leave as-is for now since we're just copying over the changes to the old polyfill.

Reply:
Yeah, I'd prefer to leave as-is for now. I'll file an issue to remove Node 12 tests at some point.

Reply:
Filed #6 to track